buildSrc/src/main/java/xyz/wagyourtail/doclet/tsdoclet/Main.java

```diff
@@ -61,7 +61,7 @@ public boolean run(DocletEnvironment environment) {
         Set<EventParser> eventClasses = new LinkedHashSet<>();
 
 
-        outputTS = new FileHandler(new File(OutputDirectory.outputDir, "JsMacros-" + Version.version + ".d.ts"));
```

Comment:
is this change actually necessary? I'd prefer not to, in order to allow users of the non-npm environment to know what version their types are for

Reply:
Just seemed unnecessary for npm, since the version is in the package itself.

TODO: Put version number back, ensure tsconfig/package.json reference it correctly

Reply:
looks like i forgot to leave a comment here - done in [e4c5229](https://github.com/JsMacros/JsMacros/pull/62/commits/e4c52298dbf958a77b5a6bde2d1d94490f6ee2ac)